docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -559,11 +559,19 @@ sources.sql_database.credentials="mssql+pyodbc://loader:loader@localhost/dlt_dat
 sources.sql_database.credentials="mssql+pyodbc://loader:loader@localhost/dlt_data?TrustServerCertificate=yes&driver=ODBC+Driver+17+for+SQL+Server"
 ```
 
-***To use long strings (>8k) and avoid collation errors**:
+**To use long strings (>8k) and avoid collation errors**:
 ```toml
 sources.sql_database.credentials="mssql+pyodbc://loader:loader@localhost/dlt_data?LongAsMax=yes&driver=ODBC+Driver+17+for+SQL+Server"
 ```
 
+**To fix MS SQL Server connection issues with ConnectorX**:
+
```

Comment:
add "Some users have reported issues with MS SQL server and connector x. The issues are outside of dlt and seem to be around how  aconnection is created. Here's a user suggested solution (Thanks @User)

Reply:
@markjames

Reply:
@markjamesm

Reply:
@dat-a-man will you add this?
"Some users have reported issues with MS SQL Server and Connector X. The problems are not caused by dlt, but by how connections are made. A big thanks to [Mark-James M](https://github.com/markjamesm) for suggesting a solution."

Reply:
updated